pyproject.toml

```diff
@@ -1,6 +1,55 @@
 [build-system]
-requires = ["setuptools>=41.0", "wheel", "setuptools_scm[toml]>=3.4"]
-build-backend = "setuptools.build_meta"
+requires = ["hatchling>=1.8.0", "hatch-vcs", "hatch-fancy-pypi-readme"]
+build-backend = "hatchling.build"
+
+[project]
+name = "bo4e-python"
+description = "Python Library that implements the BO4E Standard."
+license = { text = "MIT" }
+requires-python = ">=3.10"
+authors = [{ name = "Kevin Krechan", email = "kevin.krechan@hochfrequenz.de" }]
+keywords = ["bo4e", "bdew", "edi@energy"]
+classifiers = [
+  "Development Status :: 4 - Beta",
+  "Intended Audience :: Developers",
+  "License :: OSI Approved :: MIT License",
+  "Operating System :: OS Independent",
+  "Programming Language :: Python",
+  "Programming Language :: Python :: 3 :: Only",
+  "Programming Language :: Python :: 3.10",
+  "Programming Language :: Python :: 3.11",
+]
+dependencies = [
+  "iso3166",
+  "pydantic>=2.0.0",
+  "pyhumps",
+]
+dynamic = ["readme", "version"]
+
+[project.urls]
+Changelog = "https://github.com/Hochfrequenz/BO4E-python/releases"
+Homepage = "https://github.com/Hochfrequenz/BO4E-python"
```

Comment:
```suggestion
Changelog = "https://github.com/bo4e/BO4E-python/releases"
Homepage = "https://github.com/bo4e/BO4E-python"
```

Reply:
fixed in c317834d4614f14b56b5c17ec786fc738faa8ed2